compiler/src/iree/compiler/Dialect/Flow/Transforms/BUILD

```diff
@@ -75,6 +75,7 @@ iree_compiler_cc_library(
     ],
     deps = [
         ":PassesIncGen",
+        "//compiler/src/iree/compiler/Codegen/Utils",
```

Comment:
Please dont add dependence from `Codegen` to `Flow`. Thats a layering violation. If needed copy the utility over.